<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ce bouton sert Ã  se &lt;span style=&quot; font-weight:600;&quot;&gt;connecter&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1041</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="connexion_page">
     <widget class="QLineEdit" name="username_field">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>140</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_field">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>210</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="local_mode_button">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>280</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Mode Local</string>
      </property>
     </widget>
     <widget class="QLabel" name="username_label">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>140</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="password_label">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>210</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="user_creation_page">
     <widget class="QLineEdit" name="username_creation">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>140</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_creation">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>180</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="submit_user">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>220</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Soumettre</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_to_login_page">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>270</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Back at login page</string>
      </property>
     </widget>
     <widget class="QLabel" name="username_label_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>140</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="password_label_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>180</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="local_mode_page">
     <widget class="QLineEdit" name="query_field">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>410</y>
        <width>781</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="insert_button">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>450</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Insert</string>
      </property>
     </widget>
     <widget class="QPushButton" name="update_button">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>450</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_button">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>450</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="select_button">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>450</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
     <widget class="QPushButton" name="select_all_button">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>450</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">&lt;html&gt;&lt;bold&gt;select*&lt;/bold&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Select *</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deletion_button">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>450</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete field</string>
      </property>
     </widget>
     <widget class="QPushButton" name="send_query_button">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>510</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Query</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="creation_db_page">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>480</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>490</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>340</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>280</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1044</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
